recipes/nyxus/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
this should come from the upstream tarball, if at all possible

Reply:
The upstream actually has the license. I will remove it from here.